### DISCUSSION TITLE
Couldn't find Unittest class in the repository

### DISCUSSION THREAD

**Discussion by Ravin507** [pimcore/pimcore#11718] [Q&A / Support]
Hi Team,

We couldn't find Unittest class in the repo as it is used in many unit test files. Could you please provide the link or folder structure of the unittest class file.

For ref: use Pimcore\Model\DataObject\Unittest; - this file is not in the repo and even after clone to local.

Thanks

**Comment by jdreesen** [comment]
I think this file is generated by Codeception.

**Comment by dvesh3** [comment]
It is initialized in model helper https://github.com/pimcore/pimcore/blob/10.x/tests/_support/Helper/Model.php#L436

**Reply by Ravin507** [reply]
Hi @dvesh3 How to call Unittest in my test class as it is in Model.php. How can we use unittest class. Could you please suggest.

**Reply by dvesh3** [reply]
@Ravin507 Closing it as duplicate of #11803.
